test/tc_basic.rb

```diff
@@ -114,7 +114,7 @@ def test_set_and_get_point_from_wkt
               obj_ = klass_.new
               assert_nil(obj_.latlon)
               obj_.latlon = 'SRID=1000;POINT(1 2)'
-              assert_equal(@factory.point(1, 2), obj_.latlon)
+              assert_equal(RGeo::Cartesian.preferred_factory(srid: 1000).point(1, 2), obj_.latlon)
```

Comment:
I'm on Mac OS X 10.11, Ruby 2.3.3, MySQL 5.7

please take a look at this, debugger inside test ~ line 117

```ruby
> @factory
=> #<RGeo::Geos::CAPIFactory:0x3feb86068d9c srid=3785 bufres=1 flags=8>
> RGeo::Cartesian.preferred_factory(srid: 1000)
=> #<RGeo::Geos::CAPIFactory:0x3feb85d28100 srid=1000 bufres=1 flags=8>
> assert_equal(@factory.point(1, 2), obj_.latlon)
=> true
> assert_equal(1000, obj_.latlon.srid)
=> true
> RGeo::VERSION
=> "0.6.0"
```

You can see that factories differ in srid attribute (3785 vs 1000) but assertions return true